README.md

```diff
@@ -2,117 +2,18 @@
 
 <img src="./doc/images/overview/PT%20hero%20image.png"/>
 
-Microsoft PowerToys is a set of utilities for power users to tune and streamline their Windows experience for greater productivity. Inspired by the [Windows 95 era PowerToys project](https://en.wikipedia.org/wiki/Microsoft_PowerToys), this reboot provides power users with ways to squeeze more efficiency out of the Windows 10 shell and customize it for individual workflows.  A great overview of the Windows 95 PowerToys can be found [here](https://socket3.wordpress.com/2016/10/22/using-windows-95-powertoys/).
+Welcome to Microsoft PowerToys! **For new users, check out our newly published [Mircosoft Docs: PowerToys](https://docs.microsoft.com/en-us/windows/powertoys/) page to get started.** The content on this repo will be focused on relevant updates and resources for active contributors and developers of the various PowerToys projects. For info on [downloads & installation](https://docs.microsoft.com/en-us/windows/powertoys/install), [PowerToys guides and overviews](https://docs.microsoft.com/en-us/windows/powertoys/), or any other tools and resources for [Windows development environments](https://docs.microsoft.com/en-us/windows/dev-environment/overview), be sure to check out Microsoft Docs!
 
-For a video overview of PowerToys, including install steps and a walkthrough of the available utilities, check out the [PowerToys: Utilities to customize Windows 10](https://www.youtube.com/watch?v=F-d7KiwpnMA) episode of Tabs vs Spaces on YouTube.
-
-[Downloading & Release notes][github-release-link]   |   [What's Happening](#whats-happening)   |   [Contributing to PowerToys](#contributing)   |   [Known issues](#known-issues)
+[Known issues](#known-issues) | [What's Happening](#whats-happening) | [Core-Team Roadmap](#powertoys-roadmap) | [Contributing to PowerToys](#contributing)
```

Comment:
Change to just **Roadmap**.  Core-team is a term we use but may not make sense to everyone else.